packages/modules/viewer-react/src/components/iModel/IModelBusy.tsx

```diff
@@ -5,23 +5,23 @@
 
 import "./IModelBusy.scss";
 
-import { BodyText, LoadingSpinner, SpinnerSize } from "@bentley/ui-core";
+import { IModelApp } from "@bentley/imodeljs-frontend";
+import { ProgressBar } from "@bentley/ui-core";
```

Comment:
could we just use the one from itwinui instead? like we're going to switch very soon

Reply:
discussed internally, will just switch to itwinui in the next version